sktime/transformations/panel/signature_based/_signature_method.py

```diff
@@ -10,6 +10,8 @@
     _WindowSignatureTransform,
 )
 
+from sktime.utils.validation._dependencies import _check_soft_dependencies
+
 
 class SignatureTransformer(BaseTransformer):
     """Transformation class from the signature method.
```

Comment:
not blocking, but I notice - reading this - the docstring isn't very detailed. Perhaps you can improve it?

* the method should briefly be described, e.g., what do all the parameters do? How does this relate to the paper?
* paper reference should move to "references" section and cited
* parameters that have defaults should have defaults stated
* parameters that have specific values, like, `window_name` should have these listed and explained

As said, not blocking, but I think any of these would be an improvement.

Reply:
Did some of it as I can :)